src/main/java/org/ice4j/socket/jdk8/DelegatingSocketChannel.java

```diff
@@ -53,6 +73,37 @@ public DelegatingSocketChannel(T delegate)
 
         delegateAsSelChImpl
             = (delegate instanceof SelChImpl) ? (SelChImpl) delegate : null;
+
+        Method method;
+        try
+        {
+            method = SelChImpl.class.getMethod(
+                "translateAndSetInterestOps",
+                Integer.TYPE, SelectionKeyImpl.class);
+        }
+        catch(NoSuchMethodException e)
+        {
+            method = null;
+
+            classLogger.log(
+                Level.SEVERE,
```

Comment:
This will log a `SEVERE` level message in either Java 8 or Java 11. Maybe it would be better to only log if neither method is found?

Reply:
Yeah, you are right I saw it today ... maybe we will revert it cause it failed to work when only java11 is installed on the system, not sure why. 
Testing before that with java8 and 11 installed and 11 to be default this was working.

Reply:
Is Java11 support still work-in-progress?

Reply:
Yes, any help is welcome.